src/Features/ShippingLabelBanner.php

```diff
@@ -107,9 +107,9 @@ public function add_meta_boxes( $post_type, $post ) {
 				'normal',
 				'high',
 				array(
-					'context'               => 'shipping_label',
-					'order_id'              => $post->ID,
-					'shippable_items_count' => $this->count_shippable_items( $order ),
+					'context' => 'shipping_label',
+					'order'   => $post->ID,
+					'items'   => $this->count_shippable_items( $order ),
```

Comment:
@c-shultz I am trying to understand this part but I can't find reference to `shippable_items_count` in WCS. Is this something to do with code splitting? Or is it because of the key change here https://github.com/woocommerce/woocommerce-admin/pull/5025/files#r474927056?